gitcloner

```diff
@@ -32,5 +32,4 @@ def main():
     account.clone_repos()
 
 
-if __name__ == '__main__':
-    main()
+main()
```

Comment:
@shakib609 keep the line... keeps everything with conventions and doesn't change the behaviour